docs/installation/server-config/config-reference.rst

```diff
@@ -389,6 +389,20 @@ A list of tags to be included with emitted :ref:`JMX metrics <ksql-monitoring-an
 formatted as a string of ``key:value`` pairs separated by commas.
 For example, ``key1:value1,key2:value2``.
 
+.. _ksql-streams-state-dir:
+
+----------------------
+ksql.streams.state.dir
+----------------------
+
+Sets the storage directory for stateful operations, like aggregations and
```

Comment:
Worth adding a note that this must be unique for every ksql server running on that box? @AlanConfluent also ran into this recently .. Otherwise, the servers may appear to do be stuck without doing any work 

also the default in Streams is https://kafka.apache.org/documentation/#streamsconfigs  `/tmp/kafka-streams`. Do we override this specifically in ksql code?

Reply:
@fireinhole Do you know if we override the default state directory in ksqlDB?

Reply:
There is no override. KSQL uses the Streams default.